docs/operator-manual/applicationset/Progressive-Syncs.md

```diff
@@ -53,7 +53,7 @@ Once a change is pushed, the following will happen in order.
 
 * All `env-dev` Applications will be updated simultaneously.
 * The rollout will wait for all `env-qa` Applications to be manually synced via the `argocd` CLI or by clicking the Sync button in the UI.
-* 10% of all `env-prod` Applications will be updated at a time until all `env-prod` Applications have been updated.
+* 10% of all `env-prod` Applications will be updated at a time until all `env-qa` Applications have been updated.
```

Comment:
I believe the current docs are correct. The `env-prod` applications will be updated 10% at a time until all of them are updated.